### PR TITLE
add xCell recipe

### PR DESCRIPTION
**Pull request by grst**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Adds a recipe for [xCell](https://github.com/dviraran/xCell), a tool for estimating immune cell contents from gene expression data.

### CODE REVIEW

File: recipes/r-xcell/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+package:
+  name: r-xcell
+  version: "1.2"
+
+source:
+  url:
+    - https://github.com/dviraran/xCell/archive/1.2.tar.gz
```
Comment (bgruening) [inline recipes/r-xcell/meta.yaml]: you can use jinja variables here for the version number

File: recipes/r-xcell/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  sha256: 52dcd5cfb5c8a4870cb485ee57f34d7830b23bdc61fd3012d859fdcc22ad941c
+
+build:
+  number: 0
```
Comment (bgruening) [inline recipes/r-xcell/meta.yaml]: using script: here you can remove the build file

File: recipes/r-xcell/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - r-base =3.5.1
```
Comment (bgruening) [inline recipes/r-xcell/meta.yaml]: please do not pin r-base and also the other pins do look way to strict

Comment (grst) [reply recipes/r-xcell/meta.yaml]: done. 
As the original DESCRIPTION does not specify any version, I removed all versions here, too.

File: recipes/r-xcell/meta.yaml (hunk 2 above)
Comment (grst) [reply recipes/r-xcell/meta.yaml]: sorry, I'm not sure what you mean...

Comment (grst) [reply recipes/r-xcell/meta.yaml]: got it^^
Thanks @epruesse!

**Comment by epruesse** [comment]
LGTM

**Comment by epruesse** [comment]
@biocondabot recheck

**Comment by epruesse** [comment]
@biocondabot merge

**Pull request merged**